commands/rootCmd.go

```diff
@@ -91,4 +77,30 @@ func init() {
 	rootCfg.GKey("database.sslmode", verdeter.IsStr, "", "[DB] the sslmode to use when connecting to the database server")
 	rootCfg.SetRequired("database.sslmode")
 
+	loggerModeKey := "logger.mode"
```

Comment:
split this in different files: 
commands/database.go: func initDatabaseCommands(**) { // init all command of database}